lib/utils/RequestHandler.js

```diff
@@ -214,7 +214,7 @@ class RequestHandler {
                     return reject(new RuntimeError(error))
                 }
 
-                if (retryCount === totalRetryCount) {
+                if (retryCount >= totalRetryCount) {
```

Comment:
lets split this PR:
- bugfix for retryCount
- new feature for screenshotOnReject